src/docs/asciidoc/en/rlsnotes/rlsnotes40/_dml-40.adoc

```diff
@@ -35,6 +35,17 @@ The feature is defined in (SQL:2011): 7.6 <table reference> (Feature T491).
 .Examples
 [source]
 ----
+/* select customers with they last order date and number */
+select c.name, ox.order_date as last_order, ox.number
+from customer c
+  left join LATERAL
+  (
+  select first 1 o.order_date, o.number
+  from orders o
+  where o.id_customer=o.id
+  order by o.ORDER_DATE desc
+  ) as ox
+----
```

Comment:
`----` ends the current source block, and this breaks the rest of the document, because the remainder of the example is outside the current source block (and then starts a new one).

Replace with `--` (as used between the queries in the original example).

Reply:
I did not find markup definition yet.

Reply:
See https://docs.asciidoctor.org/asciidoc/latest/verbatim/source-blocks/

Reply:
and https://docs.asciidoctor.org/asciidoc/latest/blocks/delimited/